locksmith/package.json

```diff
@@ -14,7 +14,8 @@
     "db:migrate": "sequelize db:migrate",
     "fail-pending-changes": "../scripts/pending-changes.sh",
     "ci": "npm run lint && npm test && npm run reformat && npm run fail-pending-changes",
-    "build": "echo 'Nothing to build'"
+    "build": "echo 'Nothing to build'",
+    "tsc": "tsc"
```

Comment:
We can probably turn `build` into an alias for `tsc`?

Reply:
yup.

Reply:
That's what we did for #2076, that way locksmith's TypeScript will get built before deploy